core/controllers/library.py

```diff
@@ -333,15 +408,25 @@ class CollectionSummariesHandler(base.BaseHandler):
     """Returns collection summaries corresponding to collection ids."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'stringified_collection_ids': {
+                'schema': {
+                    'type': 'custom',
+                    'obj_type': 'ConvertStringifiedListToProperList'
```

Comment:
Discussion regarding the name of this class is already going on a PR.
Will update this name appropriately in the end.